README.md

```diff
@@ -474,6 +476,45 @@ as a representation of the following vault data:
 
 **Security Consideration** - When using the GCP IAM Auth type, ensure that the capability for the GCP SA to use the `signjwt` permission is limited only to the service accounts you wish to authenticate with to Vault. Providing your GCP SA the `signjwt` permission, such as through `iam.serviceAccountTokenCreator`, when done at the project level will over-authorize your service account to be able to sign JWTs of any other service account in the project, thus impersonating them. It is best practice to bind these permissions against the service account itself, and not at the project level. For more information, see the [GCP Documentation](https://cloud.google.com/iam/docs/granting-roles-to-service-accounts#granting_access_to_a_service_account_for_a_resource) on how to grant permissions against a specific service account.
 
+**Azure IAM Example - Writing to a File**:
+
+Assume you have the following Vault Azure Auth Role, `vault-role-name`:
```

Comment:
Possibly provide a link to https://www.vaultproject.io/api/auth/azure#create-role for first time user/admin clarity.